Data/IntMap/Internal.hs

```diff
@@ -286,7 +286,11 @@ module Data.IntMap.Internal (
 
 #if MIN_VERSION_base(4,8,0)
 import Data.Functor.Identity (Identity (..))
-import Control.Applicative (liftA2)
+import Control.Applicative (liftA2
+#if MIN_VERSION_base(4,9,0)
+                           , Const(..)
+#endif
+                           )
```

Comment:
Please don't split the import list like this for such a little thing. You can just use independent imports for the two versions.

Reply:
Done, yeah I thought that looked a little weird.